pkgs/misc/drivers/sundtek/default.nix

```diff
@@ -3,7 +3,7 @@
 with stdenv.lib;
 
 let
-  version = "28-07-2015";
+  version = "07-09-2015";
```

Comment:
Using this style of version leads to bad ordering. What about switching to `2015-09-07`, for example?